js/reference.js

```diff
@@ -2410,6 +2422,36 @@
                     this._display.sourceHasWaitFor = false;
                 }
 
+                // current default chaise uses is open for compact and close for all other contexts
+                var fpo = this._context === module._contexts.COMPACT ? true : false;
```

Comment:
It's true that in the current implementation (and most probably always) the facet will only be displayed in the `compact` contexts, but you don't need to hard code it here. The less assumption we make and the more general/simple code is, it's easier to maintain.

Also the first line is completely ignoring the config property which would make understanding this feature unnecessary harder. I don't think you should hardcode the chaise's default behavior here. The code here should just look at the context and config without any other assumptions.